openlibrary/templates/lib/nav_head.html

```diff
@@ -1,7 +1,19 @@
 $def with (page)
+$ is_privileged_user = ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians'))
 
 <header id="header-bar" class="header-bar">
-  $if ctx.user:
+  $if is_privileged_user:
+    $ loginLinks = [
+    $     { "href": ctx.user.key, "text": _("My Profile"), "track": "MainNav|MyProfile" },
+    $     { "href": homepath() + "/merges", "text": _("Pending Requests"), "class": "mr-notifications" ,"track": "MainNav|MyRequests" },
```

Comment:
I think this is the only line which needs to be added under an `is_privileged_user` section and I think instead of using the `mr-notifications` class, we can just put the number next to the text like:

```
$_("Pending Requests") "(%s)" % get_counts_mode(mode='open')